src/clusterfuzz/stacktraces/constants.py

```diff
@@ -99,6 +99,7 @@
     r'===BUG DETECTED: Arbitrary file open===')
 EXTRA_SANITIZERS_ARBITRARY_DNS = re.compile(
     r'===BUG DETECTED: Arbitrary domain name resolution===')
+EXTRA_SANITIZERS_PYSECSAN_PYSECSAN = re.compile(r'===BUG DETECTED: PySecSan:')
```

Comment:
"PYSECSAN" is repeated in the variable name?

Reply:
Bumping this comment :) The variable name seems to have some unnecessary repetition?